core/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -4,6 +4,10 @@
     <!--suppress CheckTagEmptyBody -->
     <string name="blank" translatable="false"></string>
     <string name="login">Login</string>
+    <string name="share_app_title">Share App</string>
+    <string name="share_app_description">Share App with Others</string>
```

Comment:
Text should be "Share this app with others", which would make the name "share_this_app_with_others".